Bottom: cbeb135f6af066cdfe16ffe3088c7970fc76a597
Top:    3d1b70c31ba8a1c446dec7a3e57d1855fdf25d2d
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2014-01-02 21:17:57 -0500

Refresh of zarlink

---

diff --git a/target/linux/ramips/rt305x/profiles/asiarf.mk b/target/linux/ramips/rt305x/profiles/asiarf.mk
index c15c91c..111cd80 100644
--- a/target/linux/ramips/rt305x/profiles/asiarf.mk
+++ b/target/linux/ramips/rt305x/profiles/asiarf.mk
@@ -20,6 +20,7 @@ $(eval $(call Profile,AWM002EVB))
 define Profile/ZARLINK
 	NAME:=AsiaRF Zarlink
 	PACKAGES+=kmod-usb-core kmod-usb-ohci kmod-usb2 kmod-ledtrig-usbdev \
+	kmod-sound-core \
 	kmod-i2c-core kmod-i2c-gpio kmod-sound-soc-rt5350-le89156
 endef
